packages/lending/src/protocols/radiant-v2/lending-protocol.ts

```diff
@@ -101,14 +126,44 @@ export class LendingProtocol extends Protocol {
     return this._aTokenIface;
   }
 
-  get reserves() {
-    return configMap[this.chainId].reserves;
+  private _tokensForDeposit?: common.Token[];
+
+  async getTokensForDeposit() {
+    if (!this._tokensForDeposit) {
+      const tokenList = await apisdk.protocols.radiantv2.getDepositTokenList(this.chainId);
+
+      const tokens = tokenList
+        .filter((tokens) => isTokenForDeposit(this.chainId, tokens[0]))
+        .map((tokens) => tokens[0]);
+
+      this._tokensForDeposit = tokens;
+    }
+
+    return this._tokensForDeposit;
+  }
+
+  private _tokensForBorrow?: common.Token[];
+
+  async getTokensForBorrow() {
+    if (!this._tokensForBorrow) {
+      const tokenList = await apisdk.protocols.radiantv2.getBorrowTokenList(this.chainId);
+
+      const tokens = tokenList.filter((token) => isTokenForBorrow(this.chainId, token));
+
+      this._tokensForBorrow = tokens;
+    }
+
+    return this._tokensForBorrow;
   }
 
-  getMarketName() {
+  getProtocolName() {
     return DISPLAY_NAME;
   }
 
+  getMarketName() {
+    return this.market.id;
+  }
+
```

Comment:
Why we introduce this change on abstract `protocol.ts`? And it seems it should be `getMarkeId`?

Reply:
There are three different protocols like this. We need to get the protocol name and market name to display
`getMarketName` for single market protocol like radiantv2 is not very useful for user

```
Protocol id: radiant-v2
Protocol name: Radiant V2
makret name: mainnet

Protocol id: compound-v3
Protocol name: Compound V3
makret name: USDC

Protocol id: morphoblue
Protocol name: Morpho Blue
makret name: wstETH/WETH
```